lib/validation/validators/minLength.ts

```diff
@@ -0,0 +1,5 @@
+import { helpers } from '@vuelidate/validators'
+
+export function minLength(value: string, length: number): boolean {
+  return !helpers.req(value) || value.length >= length
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70